event-exporter/sinks/stackdriver/sink_test.go

```diff
@@ -206,3 +207,14 @@ func waitWritesCount(q chan struct{}, want int) int {
 	time.Sleep(100 * time.Millisecond)
 	return len(q)
 }
+
+type monitoredResourceFactoryDummy struct {
```

Comment:
Why do you need a dummy, can you just use newSdMonitoredResourceFactory()? This way you can also skip the monitoredResourceFactory interface.